Libs/Gtk4/Core/Classes/Application.cs

```diff
@@ -1,9 +1,9 @@
-using Gio;
+using Gio.Sys;
```

Comment:
I'm not too comfortable with importing another library's `Sys` namespace like this. Ideally only the corresponding higher-level wrapper should ever access it (i.e. We'd wrap `ApplicationFlags` in an enum to avoid re-exporting `Gio.Sys`).

Maybe for now we could do `@new(..., Gio.Sys.ApplicationFlags.flags_none)` and then later refactor this out?

Reply:
You are right. I think there should be a clear distinction between the implementation and the Sys namespace. I removed all usings which where Like "using *.Sys".